File: java/classWork/PerfectNumberCheck.java
import java.util.Scanner;

public class PerfectNumberCheck{
public static void main(String...ekwe){
Scanner input = new Scanner(System.in);

System.out.print("Enter a number: ");
int number = input.nextInt();

int sum = 0;
	
for(int num = 1; num <= number /2; num++){
	if(number % num == 0){
		sum += num;
	}
}
	if(sum == number){
	System.out.print(number + " is a perfect number");
	}else{
	System.out.print(number + " is not a perfect number");
	}

}

}

File: javaScript/Task/PerfectNumberCheck.js

let number = 145;
let sum = 0;

for (let num = 1; num <= Math.floor(number / 2); num++) {
    if (number % num == 0) {
        sum += num;
    }
}
if (sum == number) {
    console.log(number + " is a perfect number");
} else {
    console.log(number + " is not a perfect number");
}